ui/pages/settings/index.scss

```diff
@@ -21,6 +21,13 @@
       @include H3;
 
       flex: 1 0 auto;
+
+      @media screen and (max-width: 575px) {
```

Comment:
Certainly nothing this PR needs to accomplish, but it would be good if we took more advantage of the breakpoints from within our design system: https://github.com/MetaMask/metamask-extension/blob/develop/ui/css/design-system/breakpoints.scss#L4

Weird though, that `break-small` is `575px` and `break-large` is `576px` xD